build.sbt

```diff
@@ -61,12 +61,12 @@ lazy val check = project
     moduleName := "rsc-check",
     scalacOptions += "-Xexperimental",
     libraryDependencies += "ch.qos.logback" % "logback-classic" % "1.2.3",
-    libraryDependencies += "io.github.java-diff-utils" % "java-diff-utils" % "4.0",
     libraryDependencies += "commons-cli" % "commons-cli" % "1.4",
     libraryDependencies += "org.ow2.asm" % "asm" % V.asm,
     libraryDependencies += "org.ow2.asm" % "asm-util" % V.asm,
     libraryDependencies += "org.scala-lang" % "scalap" % V.scala,
     libraryDependencies += "org.eclipse.jgit" % "org.eclipse.jgit" % "4.6.0.201612231935-r",
+    libraryDependencies += "io.github.java-diff-utils" % "java-diff-utils" % "4.0",
```

Comment:
I don't mind the re-order here, but what is the reason behind it?  Looks like it's moved down to be between org.scala-lang.scalap and org.scala-lang.scala-compiler.  Did you mean to sort them all?